### DISCUSSION TITLE
The repository [name] has been retired and cannot be reused.

### DISCUSSION THREAD

**Discussion by sekerhalithamza** [community/community#34568] [Repositories] [answered]
I am trying to create a readme repository for my profile but I have a error like this what should I do?

**Comment by jfullstackdev** [comment, marked as answer]
see this 

https://github.blog/2018-04-18-new-tools-for-open-source-maintainers/

there is no assurance, you might want to try requesting it 

https://support.github.com/contact/account

just don't change your username because it's part of the URL, that will really cause errors

**Reply by sekerhalithamza** [reply]
thanks
